linux-exploit-suggester.sh

```diff
@@ -1410,6 +1410,16 @@ author: blasty
 EOF
 )
 
+EXPLOITS_USERSPACE[((n++))]=$(cat <<EOF
+Name: ${txtgrn}[CVE-2017-5618]${txtrst} setuid screen v4.5.0 LPE
+Reqs: pkg=screen,ver>=4.5.0,ver<=4.5.0
```

Comment:
`doVersionComparision` supports `==`. I'm fairly certain you can use that here.

https://github.com/mzet-/linux-exploit-suggester/blob/3c113ae644b60b36c8471bc7e6a35fdf8d494adb/linux-exploit-suggester.sh#L1839-L1864

```suggestion
Reqs: pkg=screen,ver==4.5.0
```

Reply:
Yes, that's better. I just copied an example and adjusted it without looking further. Will do a PR more carefully next time.